pennylane/grouping/utils.py

```diff
@@ -658,6 +658,38 @@ def is_qwc(pauli_vec_1, pauli_vec_2):
     return True
 
 
+def are_pauli_words_qwc(lst_pauli_words):
+    """Given a list of observables assumed to be valid Pauli words, determine if they
+     are all mutually qubit-wise commuting.
+
+    This implementation has time complexity ~ O(m * n) for n wires and m Pauli words.
```

Comment:
What does `n wires` mean? No wires seem to have been mentioned/defined up until here.

Reply:
$n$ in this case refers to the maximum number of wires that must be used to describe every observable in the list. 

eg. ```obs_lst = [qml.PauliX(0), qml.PauliX(1)@qml.PauliZ(0), qml.Identity(3), qml.PauliY(2)] ```
$n = 4$ as we would need a total of 4 wires on the device to support all of these observables.